documentation/experimental_features.md

```diff
@@ -31,43 +31,41 @@ managed via Bolt commands. This key supports glob pattern for easily listing
 multiple policies, i.e. boltproject::* includes boltproject::admin and
 boltproject::sshkeys policies.
 
-The following configuration file makes the `boltproject::admin` and
-`boltproject::sshkeys` policies available to the project:
-
+The following configuration file makes the `bolt::project::admin` and
+`bolt::project::sshkeys` policies available to the project:
 ```
 ---
-name: boltproject
+name: bolt
 policies:
-- boltproject::admin
-- boltproject::sshkeys
+- bolt::project::admin
+- bolt::project::sshkeys
 ```
 
-### Policy management overview
+### Creating new policies 
+
+You can use Bolt to create a new policy in your project and add it to the
+`policies` key in your project configuration. For example, to create a new
+project-level policy named `bolt::project::user`, run:
 
-Policies can be managed through the `policy` command, with `apply`, `new` and
-`show` subcommands or the corresponding Powershell cmdlets `Invoke-BoltPolicy`,
-`Get-BoltPolicy`, and `New-BoltPolicy`. Use '-h' with any of the policy
-subcommands for more detail or `Get-Help` followed by any Powershell cmdlet
-name.
+* _*nix shell command_
+  ```
+  bolt policy new bolt::project::user
+  ```
 
-#### Creating new policies 
-> Policy names must be in the form `<PROJECT NAME>::<POLICY NAME>`. This is
-> referred to as `<POLICY NAME>` below. Policy names must also follow [class
-> naming
-> conventions](https://puppet.com/docs/puppet/7/lang_reserved.html#classes-and-defined-resource-type-names).
+* _PowerShell cmdlet_
+  ```
+  New-BoltPolicy bolt::project::user
```

Comment:
```suggestion
  New-BoltPolicy -Name bolt::project::user
```

Reply:
Was going to ask about getting the Powershell scripts in case there was an added param like this. :)